app/src/main/java/com/eis/geoCalendar/network/SMS/SMSEventNetworkManager.java

```diff
@@ -0,0 +1,91 @@
+package com.eis.geoCalendar.network.SMS;
+
+import android.content.Context;
+
+import androidx.annotation.NonNull;
+
+import com.eis.communication.network.NetworkManager;
+import com.eis.communication.network.listeners.GetResourceListener;
+import com.eis.communication.network.listeners.InviteListener;
+import com.eis.communication.network.listeners.RemoveResourceListener;
+import com.eis.communication.network.listeners.SetResourceListener;
+import com.eis.geoCalendar.gps.GPSPosition;
+import com.eis.geoCalendar.network.EventNetworkManager;
+import com.eis.smslibrary.SMSPeer;
+import com.eis.smsnetwork.SMSFailReason;
+import com.eis.smsnetwork.SMSNetworkManager;
+
+import java.util.ArrayList;
+
+/**
+ * Simple network manager for {@link SMSStringEvent}s that uses {@link SMSStringEventNetworkManagerAdapter}
+ * as network. Uses Singleton Design pattern since {@link SMSStringEventNetworkManagerAdapter} is a singleton
+ *
+ * @author Turcato
+ */
+public class SMSEventNetworkManager implements EventNetworkManager<SMSStringEvent, SMSPeer, SMSFailReason> {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A class which only has private constructors should be final](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=5077110)